pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-admin-reconciler.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-admin-reconciler
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  attributeRestrictions: null
+  resources:
+  - rolebindings
+  - clusterrolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  - network.openshift.io
```

Comment:
This is probably wrong. There is only netnamespaces in the network api group.

Reply:
Will remove.